services/spar/src/Spar/Data.hs

```diff
@@ -373,24 +379,24 @@ lookupScimToken
   => ScimToken -> m (Maybe ScimTokenInfo)
 lookupScimToken token = do
     mbRow <- retry x1 . query1 sel $ params Quorum (Identity token)
-    pure $ fmap (\(_, stiTeam, stiId, stiIdP, stiDescr) -> ScimTokenInfo{..}) mbRow
+    pure $ fmap fromScimTokenRow mbRow
   where
     sel :: PrepQuery R (Identity ScimToken) ScimTokenRow
-    sel = "SELECT token_, team, id, idp, descr \
+    sel = "SELECT token_, team, id, created_at, idp, descr \
           \FROM team_provisioning_by_token WHERE token_ = ?"
 
--- | List all tokens associated with a team.
+-- | List all tokens associated with a team, in the order of their creation.
 getScimTokens
   :: (HasCallStack, MonadClient m)
   => TeamId -> m [ScimTokenInfo]
 getScimTokens team = do
     -- We don't need pagination here because the limit should be pretty low
     -- (e.g. 16). If the limit grows, we might have to introduce pagination.
     rows <- retry x1 . query sel $ params Quorum (Identity team)
-    pure $ map (\(_, stiTeam, stiId, stiIdP, stiDescr) -> ScimTokenInfo{..}) rows
+    pure $ sortOn stiCreatedAt $ map fromScimTokenRow rows
```

Comment:
Can't cassandra do the sorting more efficiently here?

Reply:
Not without making creation time a clustering key. Which actually might not be a bad idea 🤔 @tiago-loureiro what do you think?

```diff
    void $ schema' [r|
        CREATE TABLE if not exists team_provisioning_by_team
            ( token_        text
            , team          uuid
            , id            uuid
            , created_at    timestamp
            , idp           uuid         -- optional
            , descr         text
-           , PRIMARY KEY (team, id)
+           , PRIMARY KEY (team, id, created_at)
            ) with compaction = {'class': 'LeveledCompactionStrategy'}
+             and clustering order by (created_at asc);
    |]
```

Reply:
> Which actually might not be a bad idea 

No objections here. If u want to keep a limit, bear in mind how u want to limit those cookies (do u delete the last one, etc.)? I.e., adding a created_at / expires_at might be useful indeed, I'd just try to optimise it so that deleting/overwriting cookies is straightforward. (And yeah, 16 such cookies sounds like plenty to me).

Reply:
> If u want to keep a limit, bear in mind how u want to limit those cookies (do u delete the last one, etc.)?

These are not cookies and they are not supposed to ever disappear implicitly (or be overwritten). If you've reached your limit, we're just not going to let you add a new one.